9.tcga-classify/scripts/nbconverted/visualize-results.r

```diff
@@ -214,18 +215,18 @@ top_tp53_features$ranked <- 1:nrow(top_tp53_features)
 
 panel_d_gg <- ggplot(top_tp53_features,
```

Comment:
Maybe add comments to describe what each of the panels are, or at least a header?

Reply:
I have comments as markdown headers. Do they not come through in the nbconverted script?